evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -3008,3 +3008,76 @@ def test_partial_dependence_dfs_transformer_does_not_calculate_feature_matrix(
     assert part_dep.feature_values.notnull().all()
     assert part_dep.partial_dependence.notnull().all()
     pd.testing.assert_frame_equal(part_dep, fast_part_dep)
+
+
+@pytest.mark.parametrize(
+    "nullable_y_ltype",
+    ["IntegerNullable", "AgeNullable", "BooleanNullable"],
+)
+@pytest.mark.parametrize(
+    "grid_resolution",
+    [2, 10],
+)
+@pytest.mark.parametrize(
+    "X_has_nans",
+    [True, False],
+)
+def test_partial_dependence_with_nullable_types(
```

Comment:
Added this test because `mquantiles`, which we use in `_grid_from_X`, has a nullable type incompatibility when null values are present in `Int64` or `boolean` data. 

This isn't actually going to be problematic, though, because the incompatibility is only when null values are present in  the data, and remove any nans prior to passing the data into `mquantiles` https://github.com/alteryx/evalml/blob/main/evalml/model_understanding/_partial_dependence_utils.py#L178-L185. But I wanted to add this test in case we ever changed how we use `mquantiles`.